src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -51,106 +51,34 @@
-                .initFields(fields)
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewFlattened(context, rootNode);
-  }
-
-  public InitCodeView generateRequestObjectInitCode(MethodTransformerContext context) {
-    InitCodeNode rootNode =
-        InitCodeNode.createTree(
-            InitTreeParserContext.newBuilder()
-                .table(new SymbolTable())
-                .rootObjectType(context.getMethod().getInputType())
-                .initValueConfigMap(createCollectionMap(context))
-                .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewRequestObject(context, rootNode);
-  }
-
-  public InitCodeView generateFlatteningTestInitCode(
-      MethodTransformerContext context,
-      Iterable<Field> fields,
-      SymbolTable table,
-      TestValueGenerator valueGenerator) {
-    InitCodeNode rootNode =
-        InitCodeNode.createTree(
-            InitTreeParserContext.newBuilder()
-                .table(table)
-                .valueGenerator(valueGenerator)
-                .rootObjectType(context.getMethod().getInputType())
-                .initValueConfigMap(createCollectionMap(context))
-                .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
-                .initFields(fields)
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewFlattened(context, rootNode);
-  }
-
-  public InitCodeView generateRequestObjectTestInitCode(
-      MethodTransformerContext context, SymbolTable table, TestValueGenerator valueGenerator) {
-    InitCodeNode rootNode =
-        InitCodeNode.createTree(
-            InitTreeParserContext.newBuilder()
-                .table(table)
-                .valueGenerator(valueGenerator)
-                .rootObjectType(context.getMethod().getInputType())
-                .initValueConfigMap(createCollectionMap(context))
-                .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewRequestObject(context, rootNode);
-  }
-
-  public InitCodeView generateMockResponseObjectInitCode(
-      MethodTransformerContext context, SymbolTable table, TestValueGenerator valueGenerator) {
-    ArrayList<Field> primitiveFields = new ArrayList<>();
-    for (Field field : context.getMethod().getOutputMessage().getFields()) {
-      if (field.getType().isPrimitive() && !field.getType().isRepeated()) {
-        primitiveFields.add(field);
-      }
+  /** Generates initialization code from the given InitCodeTransformerContext object. */
+  public static InitCodeView generateInitCode(InitCodeTransformerContext context) {
```

Comment:
Why are you switching to static functions? All of the other transformers use instance methods.